notebooks/timeframes.ipynb

```diff
@@ -0,0 +1,747 @@
PC2SpdeOsau2Teq4FQpXJLO4mnvf2sPgYH+emjuA8OCBCESHX6EXT4ukss7Ah3tPa2SxoifszS6l0Sh12Z/kfJzuluBEyQkARvUd5TLarqZrL+3iqnoWrtmNEII3F00iop8/VwxL6FR3QnhvxocHs2ZHJvMnD8Wtg1rv7qDOs+1JzSjB3U0Q3KuYEyVlmvhsDk53x11cU0xxTbFLabuarj20axuM3PNWGgUVdbyxIIGIfv5m694zLZKskhq+O1ZoVZvUebY9qRklxIYFUWso08xnc3C6wK1Q9BSjSfLQu/tJP13GK7ePZ0J4b4u2vyZ6IKFBPqzanmkjCxW2oKbBwE+ny3Q3TVl7qMCtUJyHlJKnPz3Mt0cLeHpONDOjB1q8Dw93NxZMjSAlo4TD+Wrog6PgKPltUIFbobiAf23L4O3UbO6/bBh3T4no9n5uTwzH19OdNTuyrGabwrakZpTg4SZIGGrZNywtcLrA7eXuhZe7l0tpu5qurbQ/OZDH818dY3ZcKI9fM7pHukF+ntyaMJhPD+RTWFlnFfvUebYtKadKGDc4CH9vD019NgdVx61Q0PRPe/fqXUwI783aeybh7dHzQR+ZxdVMf3Erv50xkv+6Wp/VCYomqusNxD2zmeTLhvFYBx/atkbVcSsUFnCioJLkt9OI6OvPyvkJVgnaAJH9/Jkxuj/rUrOpa54lR6FP0rJLMZgcI78NThi4jxYd5WjRUZfSdjVda2oXVNSxcPVufDzdWbMokSC/zqepslT3nmmRlFQ38OmB/J6aqs6zDWnNb0f0tqtud3G6ATildaUup+1qutbSrqxrZOGaPZTXNrL+/ikM7u1ndd0pw/syemAgq3dkcmvCYITo/oAcdZ5tR2pGCXFDgvHz8rCrbndxujtuhcIcGo0mfr1uHycKKvnnXROJCQuyiY4QgsXTIjl2tpKdp0psoqHoGVX1Bg6eLmfyMP3N5t4RKnArXA4pJUs3pPPjyWKeuzmWy0fZdqq9OXGh9AvwUgNydEpa1jmMJsmUYf20NsVszJ0BJ1gI8aEQ4pgQ4qgQYoqtDVMobMVL357kw72neXjGSH6ZMMTmej6e7sy7ZCjfHSsko0hNtKA3UjPO4ekumDA0WGtTzMbcO+6Xga+llKOBOEC3WXsfDx98PHxcStvVdHui/f6eXF7ecpJbJw7mkatG2k33rslD8XJ369GAHHWebUNKRglxg/+T37aXbk/oso5bCBEEHACGSTOLvlUdt0KP/HCiiMVv7mHq8L6sXpiIp7t9M4VLPviJzw+eIXXpjC6rVxT2obKukfg/f8OvrxjO72faZpJnc7F2HXckUASsEULsF0K80Tx5cFvRZCFEmhAiraioyEKTFQrbciivnF+/s5eoAYG8dtdEuwdtgEVJkdQ2Gnl3j75mDHdl0rJLMTpQ/XYL5ly9HsAE4DUp5XigGnii7UpSypVSygQpZUJIiG0f9nTG4cLDHC487FLarqZrqfbp0hoWvbmHYD8v1ixKJKAHM9P0xOexob2YOrwvb+3MotFosqt2T3CU89wdUk+V4OXudlEHSC19NgdzAvdp4LSUclfz6w9pCuS6pLy+nPJ6bTqyaaXtarqWaJfXNNVq1zUaWbMokQG9epa37KnPi5MiOVNex9eHztpdu7s4wnnuLqkZJcQPCcbX68LRslr6bA5dBm4p5VkgVwjRkgCaARyxqVUKhRWoNxi57+00ckpqWDk/gVEDArU2iStH9yeyn78qDdQBlXWNpOc5Vv12C+Ym+h4C1gkhDgLxwF9tZ5JC0XNMJsnv3/+J3ZnnePGXcbppju/mJliUFMGB3DL2Zut7dJ6zsyfrHCYJk3VybViCWYFbSnmgOX89Tko5V0qprjiFrnnh62N8fvAMT1w7mjlxoVqbcwG/mDCYXj4erN6h7rq1JDXjXLv5bUfA6XqV+HteVPDi9NquptuV9tqULP61LYP5k4dy/2XD7KZr9j68PbhjUjhvbM8kr6yWsGBfu2l3B72e556ScqqE+PBgfDwv7gappc/moPpxK5yKTYfP8sA7e7lqzABev2si7lacZd2a5JXVctnfvufeaZEsvW6M1ua4HOW1jYz/82YeunIkv9NJr3TVj1vhkuzLKeW37+4nbnAwr9w+XrdBGyAs2JdrYgby7905VNcbtDbH5UhryW87WP12C04XuH86+xM/nf3JpbRdTbc97aziau59K42BQT6sWpBwUXmXrXR7wj3TIqmsM/Dh3tN217YEPZ1na5FyqgQvDzfGh7ffn0RLn83B6XLc1Y3VLqftarpttUuq6lmwZjcAby2aRN8Ab7vo9pQJ4b2JHxLMmh2ZzJ88FLcuviG4+nm2JqmZJUzoIL9tS11r4XR33ArXorbByD1vpVFQUccbCxKI6Kfvh0ptuWdaJFklNXx3rFBrU1yG8tpGDudXOGyaBJzwjlvh/KxLX8fvN/2egqoihsq/IOrH8fr8BIcs67omZiCDgnxYvSOTq8YO0Nocl2B35jmkA+e3Qd1xKxyMdenrSP4smYKqAgINc6A+jgqf1RSbtmhtWrfwdHdjwdQIdp4q4Uh+hdbmuASpGSV4e7gRP8Rx+m+3xekCd5B3EEHetpmGSq/arqS7bMttM6YjAAAgAElEQVQyahpr8DdcRYDxaso9PuKc+JhlW5bZRd8WPt+RGI6vp3uXA3Jc6TzbUjs1o4QJ4b07zG/bSteaOF2qJLp/tMtpu5JuTnkOfobL6Gd4hGr3HyjzeLN1uT2whc9Bfp7cMnEw6/fk8vg1owkJbP/hqiudZ1tpl9U0cORMBY/M6Lx2W0ufzcHp7rgVzk2475X0a/wddW7pFHv+HUTTALLwoHCNLesZi5IiaDCaeCc1W2tTnJqW/LZeetd0F6cL3PvO7GPfmX0upe0quicLKvGpehijWwGFXs+AaBq44ufpx/IZy+1ig618HhYSwIzR/XknNZu6RqNdtbvCmf6nUprz23FDOk+DaOmzOThd4K4z1FFn
```

Comment:
Didn't we decide to *only* use period average data?

Reply:
(I also think we should avoid flux/state variables as a SCM-specific terminology that actually defines data as by their meaning rather than by how they are specified regarding timeframes)

Reply:
> Didn't we decide to _only_ use period average data?

Sorry yes true. We have a function to convert point data to period average data right? In that case maybe 'standard for OpenSCM' --> 'way all timeseries are internally stored in OpenSCM'.

> (I also think we should avoid flux/state variables as a SCM-specific terminology that actually defines data as by their meaning rather than by how they are specified regarding timeframes)

I don't fully understand this. The one thing I would say is that state/flux variables is not SCM-specific, it's Physics.

Reply:
You're right, sorry. I just meant, one could in principle give flux variables as point data and state variables as period averages... will include the conversion functions today ;)